src/mockServer/server.js

```diff
@@ -1,62 +1,64 @@
-import XHRAdapter from "@pollyjs/adapter-xhr";

```

Comment:
What's with this change? did you change [file encoding](https://en.wikipedia.org/wiki/Character_encoding) or [line-ending](https://en.wikipedia.org/wiki/Newline) a.k.a CRLF?

(or am I worrying too much 😅?)

Reply:
I don't know why git is considering the whole file content has changed. I didn't change file encoding or line-ending. So if there's no problem related to this behavior, we're good to go.